integration/setup.go

```diff
@@ -546,9 +557,23 @@ func (ts *testSetup) close() {
 	}
 }
 
+func (ts *testSetup) mustSetTickMinimumInterval(tickMinInterval time.Duration) {
+	runtimeMgr := ts.storageOpts.RuntimeOptionsManager()
+	existingOptions := runtimeMgr.Get()
+	newOptions := existingOptions.SetTickMinimumInterval(100 * time.Millisecond)
```

Comment:
Hm, should this be `.SetTickMinimumInterval(tickMinInterval)` instead?  Don't see `tickMinInterval` used anywhere else otherwise.

Reply:
oh yeah good call, I just always pass 100 * time.Millisecond so it worked but thats not what it should do. Fixed.